lib/html/pipeline/sanitization_filter.rb

```diff
@@ -72,7 +72,10 @@ class SanitizationFilter < Filter
         },
         :protocols => {
           'a'   => {'href' => ANCHOR_SCHEMES},
-          'img' => {'src'  => ['http', 'https', :relative]}
+          'img' => {
+            'src'      => ['http', 'https', :relative],
+            'longdesc' => ['http', 'https', :relative]
```

Comment:
If `longdesc` is for images only, should the attribute whitelist be moved to the 'img' specific Hash key?